Source/Events.Processing/Projections/CompareProjectionDefinitionsForAllTenants.cs

```diff
@@ -103,4 +104,47 @@ bool EventsAreEqual(ProjectionDefinition newDefinition, ProjectionDefinition old
 
         return true;
     }
-}
\ No newline at end of file
+
+    bool CopiesAreEqual(ProjectionDefinition newDefinition, ProjectionDefinition oldDefinition, ref ProjectionDefinitionComparisonResult result)
+        => CopyToMongoDBsAreEqual(newDefinition, oldDefinition, ref result);
+    
+    bool CopyToMongoDBsAreEqual(ProjectionDefinition newDefinition, ProjectionDefinition oldDefinition, ref ProjectionDefinitionComparisonResult result)
```

Comment:
Send in the specific thing to check, here the _.Copies.MongoDB

Reply:
Done